docs/install-antora.md

```diff
@@ -1,16 +1,39 @@
 # Install Antora
 
-Installing Antora on your local development (virtual) machine doesn't take *too* much time, but there are a number of tools to install.
-If you just want to make text changes, then installing the Asciidoc Live Preview Plugin may be sufficient for your needs.
+Installing Antora on your local development (virtual) machine doesn't take *too* much time, based on whether you install the tools directly, or use the official Antora Docker container.
 
+**Please note:** if you just want to make text changes, then installing [the AsciiDoc Live Preview Plugin](https://asciidoctor.org/docs/editing-asciidoc-with-live-preview/) may be sufficient for your needs.
+
+## Use The Docker Container
+
+If you’re looking for the simplest path to making Antora available, so that you can build the documentation and preview the documentation locally then, assuming that you have Docker already installed, run the following two commands.
```

Comment:
strange apostrophe here: `'` vs `’`

Reply:
That one snuck through. I need to improve my git hooks to catch these.